fuel_test/env_attributes.yaml9

```diff
@@ -9,3 +9,102 @@ editable:
       versions:
       - password:
           value: xenroot
+  provision:
+    packages:
+      label: Initial packages
+      type: textarea
+      value: 'acl
+
```

Comment:
there are new lines in the yaml file downloaded via fuel cli command. So it's just to keep the the same layout.